Complet main with some documentation

HenrijetteOest opened the pull request:
I began some docummentation and clean up of irrelevant comments and code.

HenrijetteOest committed:
cleaning Program

HenrijetteOest committed:
cleaning and formatting code,
Left createcheepsmethod, I think we could use it maybe?

HenrijetteOest committed:
updated documentation for AuthorRepository

HenrijetteOest committed:
started documentation of cheeprepo and updated authorRepo returns in summary not finish there

HenrijetteOest committed:
documentation for ChirpCore

tomat-suppe commented on src/ChirpCore/DTOs/AuthorDTO.cs:
```diff
@@ -1,4 +1,8 @@
-	public record AuthorDTO (int Id, string Name);
+	/// <summary>
+	/// AuhtorDTO is used to transfer data of an author to our Chirp! Application.
+	/// Ensures that author(s) are unimmutable and sensitive data of our author is not exposed.
```
Is this supposed to be just 'immutable'? "Unable to be changed"

HenrijetteOest replied:
godt set, ja haha

tomat-suppe commented on src/ChirpCore/DTOs/CheepDTO.cs:
```diff
@@ -1,5 +1,13 @@
 {
+	/// <summary>
+	/// CheepDTO is used to transfer data of cheep(s) to our Chirp! Application.
+	/// Ensures that cheep(s) are unimmutable and sensitive data of our cheep is not exposed.
```
Same as previous comment, is this simply 'immutable', i.e. unable to be changed?

HenrijetteOest replied:
godt set, ja

HenrijetteOest committed:
Update AuthorDTO.cs

spelling mistake

HenrijetteOest committed:
Update CheepDTO.cs

addition text and a spelling mistake

tomat-suppe commented on src/ChirpInfrastructure/ChirpDBContext.cs:
```diff
@@ -19,27 +19,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
 		modelBuilder.Entity<Author>().HasIndex(a => a.UserName).IsUnique();
 		modelBuilder.Entity<Author>().HasIndex(a => a.Email).IsUnique();
-		//Below method structure from ChatGpt
```
should we not keep this line, or is it refering to removed code?

HenrijetteOest replied:
done

HenrijetteOest committed:
Update ChirpDBContext.cs

to be sure I added it again

HenrijetteOest committed:
Update CheepRepository.cs

added mehtod from privatetimelineOrder branch, which fixed the ordering of the cheeps 
Co-Authored: Sophie <slak@itu.dk>

tomat-suppe committed:
Merge branch 'main' into completMain

tomat-suppe committed:
rewert to old commit to complete merge with main.

tomat-suppe reviewed (approved):
slay, good work on fixing up the last stuff and adding comments!
